module/purpose/Soccer/src/Soccer.cpp

```diff
@@ -27,13 +27,15 @@
 #include "Soccer.hpp"
 
```

Comment:
import localisation/ball etc. and find the half of the field the ball is on

With will grab last version of the msg